client/pages/Assessment/Assessment.tsx

```diff
@@ -90,6 +91,7 @@ const Assessment: React.FC = () => {
           )}
         />
       </Switch>
+      <MessageCenter />
```

Comment:
the message center needs all query params from url. adding it here will brake the review .
it's needed to think to another solution

Reply:
#1406

Reply:
Find the solution . What else should you do?
Add the message center to Fra home?

Reply:
After further testing, it is working as expected in `Assessment.tsx` as verified with @rfontanarosa